src/global.test.ts

```diff
@@ -123,7 +130,9 @@ async function startFuncProcess(appPath: string): Promise<void> {
 
     childProc.stdout?.on('data', (data: string | Buffer) => {
         data = data.toString();
-        process.stdout.write(data);
+        if (!funcCliSettings.hideOutput) {
```

Comment:
Would it be clearer to name this setting `showOutput`?

Reply:
My general philosophy is to make the default value false (to match `undefined` which is false-y) and then choose the name based on that. So I couldn't do `showOutput` since then the default would have to be true

Reply:
I see. Okay, no problem. I'll try to match that approach.